gmcs/linglib/adnominal_possession.py

```diff
@@ -377,15 +377,15 @@ def customize_poss_rules(strat,mylang,ch,rules,hierarchies):
             mylang.add(phrase_rule+' := [ SYNSEM.LOCAL.CAT.VAL.SPR < >,\
                                           HEAD-DTR.SYNSEM.LOCAL [ CAT [ VAL.SPR <[ ]> ],\
                                                                   CONT.HOOK.INDEX.COG-ST uniq-id  ],\
-                                          C-CONT [ RELS <! '+POSS_REL+',\
-                                                           '+POSSESSUM_EXIST_REL+' !>,\
-                                                   HCONS <! qeq & [ HARG #harg, LARG #lbl] !> ] ].')
+                                          C-CONT [ RELS.LIST < '+POSS_REL+',\
+                                                           '+POSSESSUM_EXIST_REL+' >,\
+                                                   HCONS.LIST < qeq & [ HARG #harg, LARG #lbl]  > ] ].')
 
         elif strat.get('mod-spec')=='mod':
             mylang.add(phrase_rule+' := [ SYNSEM.LOCAL.CAT.VAL.SPR #spr,\
                                           HEAD-DTR.SYNSEM.LOCAL.CAT [ VAL.SPR #spr & <[ ]>],\
-                                          C-CONT [ RELS <! '+POSS_REL+' !>,\
-                                                   HCONS <! !> ] ].')
+                                          C-CONT [ RELS.LIST < '+POSS_REL+' >,\
+                                                   HCONS.LIST < > ] ].')
```

Comment:
After discussion with Olga, we think it could be a good idea to remove the 0-alist type and related types.  They don't make the code more succinct or more readable, but they can cause issues if you want to create other subtypes (which can be useful in other cases).

So I would suggest keeping this, and changing other instances of `HCONS 0-alist` to `HCONS.LIST <>`.

Reply:
I will replace all 0-alists with LIST < > but not now. This PR is to make sure the lists are working the same way the diff-lists were (or better).